.github/workflows/deploy.yml

```diff
@@ -0,0 +1,33 @@
+name: DR2 Deploy Ingest CC Notifications Handler Lambda
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: choice
+        description: 'Environment'
+        required: true
+        options:
+          - intg
+          - staging
+          - prod
+        default: 'intg'
+      to-deploy:
+        description: 'Version to deploy'
+        required: true
+
+permissions:
+  id-token: write
+  contents: write
+jobs:
+  deploy:
+    uses: nationalarchives/dr2-github-actions/.github/workflows/lambda_deploy.yml@main
+    with:
+      lambda-name: ${{ github.event.inputs.environment }}-ingest-cc-notification-handler
```

Comment:
The lambda is called `${{ github.event.inputs.environment }}-dr2-ingest-cc-notifications` in AWS.

Reply:
Ah, I see. I didn't name this; should I rename the lambda since `dr2-ingest-cc-notifications` is a bit vague

Reply:
I'd leave the lambda name as is. We don't have `handler` or `runner` or anything like that on the other lambdas and this repo will (probably) eventually end up in a mono-repo somewhere.

Reply:
I read back my last comment and it's not very clear. If you change the `lambda-name` in the deploy.yml file to `${{ github.event.inputs.environment }}-dr2-ingest-cc-notifications` and leave the existing name in terraform.